tests/functests/test_suite_test.go

```diff
@@ -131,10 +132,18 @@ func init() {
 		defaultWindows2k22ContainerDisk, "Windows Server 2022 container disk used by functional tests")
 	flag.StringVar(&windows2k25ContainerDisk, "windows-2k25-container-disk",
 		defaultWindows2k25ContainerDisk, "Windows Server 2025 container disk used by functional tests")
+	flag.StringVar(&preferenceArch, "preference-arch", "", "Architecture to test preferences for")
 	flag.DurationVar(&windowsReadyTimeout, "windows-ready-timeout",
 		defaultVMReadyTimeout, "Duration after Windows VM will timeout")
 }
 
+func getClusterArch(virtClient kubecli.KubevirtClient) string {
+	nodes, err := virtClient.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
```

Comment:
Would it make sense to check if the cluster is multi-architecture and if yes then print some warning?

Reply:
For now we'd like to keep it simple. We can extend later if required.